app/Http/Controllers/FaqController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\FaqModel;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class FaqController extends Controller
+{
+
+    protected $table = 'faq';
+    public function faq():View {
```

Comment:
When you follow Laravel naming conventions around Controller method naming, you can save a lot of development time.